README.md

```diff
@@ -192,13 +195,17 @@ df = pd.read_csv("my_dataset.csv")
 #### Modin is a DataFrame designed for datasets from 1MB to 1TB+ 
 
 We have focused heavily on bridging the solutions between DataFrames for small data 
-(e.g. pandas) and large data. Often data scientists require different tools for doing
-the same thing on different sizes of data. The DataFrame solutions that exist for 1KB do
-not scale to 1TB+, and the overheads of the solutions for 1TB+ are too costly for 
-datasets in the 1KB range. With Modin, because of its light-weight, robust, and scalable
-nature, you get a fast DataFrame at small and large data. With preliminary [cluster](https://modin.readthedocs.io/en/latest/using_modin.html#using-modin-on-a-cluster)
-and [out of core](https://modin.readthedocs.io/en/latest/out_of_core.html)
-support, Modin is a DataFrame library with great single-node performance and high
+(e.g. pandas) and large data. Often data scientists have to use different tools 
+for operating on datasets of different sizes. This is not only because processing large 
+dataframes is slow, but also pandas does not support working with dataframes that don't fit 
+into the available memory. As a result, pandas workflows that work well 
```

Comment:
```suggestion
for operating on datasets of different sizes. Processing large datasets with pandas is slow, and pandas does not support working with dataframes that are too large to fit into the available memory. As a result, pandas workflows that work well 
```

I don't see a need to emphasize "not only." Modin should help with both problems.